src/main/java/pl/betoncraft/betonquest/conversation/InventoryConvIO.java

```diff
@@ -281,7 +281,7 @@ public void run() {
     }
 
     @SuppressWarnings("deprecation")
-    protected SkullMeta setSkullMeta(final SkullMeta meta) {
+    protected SkullMeta updateSkullMeta(final SkullMeta meta) {
```

Comment:
This is an API breaking change.

Reply:
This is not an API break, because this method was newly implemented: https://github.com/BetonQuest/BetonQuest/commit/96b16f0a1b070be35c65e243889c9166ca242a9a#diff-68d8ec8ba4a6ff089ad344b5c0f6ef59